### DISCUSSION TITLE
Countdown Pumpkin Party

### DISCUSSION THREAD

**Discussion by IvanSkazkin** [sunflower-land/sunflower-land#2267] [Ideas]
I don't live in the USA. And I don't know what time it is. You can add a countdown to how much time is left before the event.

![5 1](https://user-images.githubusercontent.com/103871467/219980560-e36eba7d-c88b-4f20-83ed-7735e5751d91.png)
![5 2](https://user-images.githubusercontent.com/103871467/219980805-dc251d7e-3af8-457d-9b04-e96df8174d21.png)

**Comment by TheDcol** [comment]
![image](https://user-images.githubusercontent.com/108110065/219982332-a3e62048-6e43-482c-83ca-504448984c34.png)


^ That is your local time I believe

For myself being in Australia it is at 3.30pm-4.30pm, for example

**Reply by IvanSkazkin** [reply]
Well if that's the case. But I have not seen such remarks here that the time is indicated with your PC or monitor. This is misleading.
